examples/Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: clean coldstart mrproper up
+all: up
+
+lisk_net := $(shell grep ^ENV_LISK_NETWORK= .env |tail -n 1 |awk -F = '{ gsub("net$$", ""); print $$2 }')
+ifeq ($(lisk_net),)
+$(error .env file does not exist or ENV_LISK_NETWORK variable is not set.)
+endif
+
+%blockchain.db.gz:
+	curl --output $(lisk_net)_blockchain.db.gz https://downloads.lisk.io/lisk/$(lisk_net)/blockchain.db.gz
+
+up:
+	docker-compose up -d
+
+coldstart: $(lisk_net)_blockchain.db.gz up
+	docker-compose stop lisk
+	docker-compose start db
+	docker-compose run --rm task dropdb --if-exists lisk
```

Comment:
Database name should come from `ENV_LISK_DB_DATABASE` which can be founded inside the `.env` file.